incubator/redash/templates/_helpers.tpl

```diff
@@ -37,8 +37,8 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "redash.adhocWorker.fullname" -}}
-{{- if .Values.fullnameOverride -}}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- if .Values.adhocWorker.fullnameOverride -}}
```

Comment:
Do you really need separate fullnameOverride for the components and configurable component names? How about doing something like this:

```
{{- define "redash.adhocWorker.fullname" -}}
{{- template "redash.fullname" . -}}-adhoc-worker
{{- end -}}
```

Reply:
I don't need this personally - I was trying mainly to keep things consistent with the Helm 2.8+ [create template](https://github.com/kubernetes/helm/blob/master/pkg/chartutil/create.go#L272) and also copying the pattern used by some of the other multi-deployment charts like [Prometheus](https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/_helpers.tpl).

If you think it's cleaner to drop fullnameOverride I am happy to remove it. If so let me know if you think fullnameOverride should be removed from all deployments or kept on the primary/web deployment.

Reply:
I'd suggest you follow the example above. You'll have one fullnameOverride affecting all derived fullnames.